qa/rpc-tests/sc_csw_eviction_from_mempool.py

```diff
@@ -0,0 +1,299 @@
+
+            csw_proofs1.append(cswMcTest.create_test_proof(
+                "sc1", sc_csw_amount, str(scid1_swapped), nullifiers1[i], pkh_mc_address,
+                ceasingCumScTxCommTree1, actCertData1))
+
+            csw_proofs2.append(cswMcTest.create_test_proof(
+                "sc2", sc_csw_amount, str(scid2_swapped), nullifiers2[i], pkh_mc_address,
+                ceasingCumScTxCommTree2, actCertData2))
+        
+        sc_csws = []
+
+        for i in range(MAX_NUM_OF_CSW_INPUTS_PER_SC):
+            sc_csws.append(
+            {
+                "amount": sc_csw_amount,
+                "senderAddress": csw_mc_address,
+                "scId": scid1,
+                "epoch": 0,
+                "nullifier": nullifiers1[i],
+                "activeCertData": actCertData1,
+                "ceasingCumScTxCommTree": ceasingCumScTxCommTree1,
+                "scProof": csw_proofs1[i]
+            })
+            sc_csws.append(
+            {
+                "amount": sc_csw_amount,
+                "senderAddress": csw_mc_address,
+                "scId": scid2,
+                "epoch": 0,
+                "nullifier": nullifiers2[i],
+                "activeCertData": actCertData2,
+                "ceasingCumScTxCommTree": ceasingCumScTxCommTree2,
+                "scProof": csw_proofs2[i]
+            })
+
+        # recipient MC address, the same for all csw
+        taddr_2 = self.nodes[2].getnewaddress()
+        sc_csw_tx_outs = {taddr_2: Decimal(sc_csw_amount*10)}
+
+        rawtx = self.nodes[0].createrawtransaction([], sc_csw_tx_outs, sc_csws)
+        funded_tx = self.nodes[0].fundrawtransaction(rawtx)
+        sigRawtx = self.nodes[0].signrawtransaction(funded_tx['hex'], None, None, "NONE")
+        finalRawtx = self.nodes[0].sendrawtransaction(sigRawtx['hex'])
+        mark_logs("sent csw tx1 {}".format(finalRawtx), self.nodes, DEBUG_MODE)
+        self.sync_all()
+
+        mark_logs("Check tx is in mempool...", self.nodes, DEBUG_MODE)
+        assert_true(finalRawtx in self.nodes[2].getrawmempool())
+
+        # send another csw input for scid1 and verify it is rejected
+        sc_csw = [{
+            "amount": sc_csw_amount,
+            "senderAddress": csw_mc_address,
+            "scId": scid1,
+            "epoch": 0,
+            "nullifier": nullifiers1[-1],
+            "activeCertData": actCertData1,
+            "ceasingCumScTxCommTree": ceasingCumScTxCommTree1,
+            "scProof": csw_proofs1[-1]
+        }]
+
+        rawtx = self.nodes[0].createrawtransaction([], sc_csw_tx_outs, sc_csw)
+        funded_tx = self.nodes[0].fundrawtransaction(rawtx)
+        sigRawtx = self.nodes[0].signrawtransaction(funded_tx['hex'], None, None, "NONE")
+        mark_logs("Sending a further csw input to scid1, expecting failure...", self.nodes, DEBUG_MODE)
+        try:
+            finalRawtx2 = self.nodes[0].sendrawtransaction(sigRawtx['hex'])
+            assert(false)
```

Comment:
Typo, `false` is not defined.

```suggestion
            assert(False)
```

Reply:
Done